recipe/meta.yaml

```diff
@@ -58,3 +57,4 @@ extra:
     - dtip
     - iainrussell
     - shahramn
+    - xylar
```

Comment:
I'm fine with *not* being a maintainer, I just figured it was friendlier to add myself in case you could use another hand.